components/navbar.js

```diff
@@ -0,0 +1,75 @@
+import { useTheme } from "next-themes";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+
+export default function Navbar() {
+    const { theme, setTheme, resolvedTheme } = useTheme();
+    const [mounted, setMounted] = useState(false);
+
+    useEffect(() => {
+        setMounted(true);
+    }, []);
+
+    return (
+        <div className>
+            <nav class="sticky flex items-center justify-between flex-wrap p-6 items-center justify-between">
+                <div class="flex items-center flex-shrink-0 text-white">
+                    <h1 className="text-semibold text-center text-4xl text-red-400">
+                        <Link href="/">BPHC Ultimate</Link>
+                    </h1>
+                    <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 1224 1584"
+                        width={32}
+                        height={41}
+                        className="text-red-400 ml-4"
```

Comment:
Use this fill color instead of text color - [https://tailwindcss.com/docs/fill](https://tailwindcss.com/docs/fill)

Reply:
corrected this